servers/src/main/java/tachyon/master/journal/JournalWriter.java

```diff
@@ -290,11 +291,15 @@ public synchronized void flush() throws IOException {
    * handles rotating full log files, and creating the next log file.
    */
   private class EntryOutputStream implements JournalOutputStream {
-    private DataOutputStream mOutputStream;
+    /** The direct output stream created by {@link UnderFileSystem} */
+    private OutputStream mOs;
```

Comment:
I think these variable names are a bit too concise. How about "mRawOutputStream" (or maybe mUfsOutputStream?) and "mDataOutputStream"?

The idea is that the DataOutputStream wraps the RawOutputStream.